Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 55 connect backend and frontend via api call

## Related ticket:
[Jira ticket SCRUM-55](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-55?atlOrigin=eyJpIjoiZDg3Mzk1ZTc2NmM0NDJjMWEyZGNiNDZjY2FhZGVmMDEiLCJwIjoiaiJ9)

## In this PR:
- Write a function to fetch all api links from Drupal
- Destructure and type the response

## Testing instruction
- Checkout branch
- Import test db in backend: `lando db-import dump.gz`
- Reinstall frontend dependencies: `npm i`
- create a `.env.local` file in the root of Frontend. add the Drupal url api like this
```json
VITE_DRUPAL_URL=http://DEFAULT_URL/api/
```
Remember to change DEFAULT_URL to your Drupal port
_**Requires CORS plugin to allow request from Backend**_

- Open frontend page and check the console. It should show the fetched links from Drupal like this:
<img width="1154" alt="image" src="https://github.com/user-attachments/assets/3dfde328-0170-4b40-aff2-2e017498391d">
